### ISSUE TITLE
[REC] VDH Fracture usage change request

### ISSUE THREAD

**Issue by dubudevs** [Hekili/hekili#3942]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Spec

DH - Vengeance

### Describe the Issue

Here are some situations where fracture is being misrecommended:

https://pastebin.com/ecuMnpyF - Why is fracture not being recommended earlier, instead opting to send 2 soulless sunders?
https://pastebin.com/j6YfeDMg - Why is fracture not being recommended at some point during the cleave spam?
https://pastebin.com/ametAUL2 = Why IS fracture being recommended here with 5 souls? 

TLDR on fracture usage is you should be pressing it any time you can without capping souls or fury, after immo aura, fel dev, and spirit bomb if you have 3-4+ souls depending on if you are in meta.

This is one of your main soul generators and souls are your main mitigation, and right now the addon is generating very very few. I am almost certain this is terrible for dps too. Recommendations seem fundamentally flawed right now, the spec feels almost unplayable using the addon's recommendations.

### How to Reproduce

Play VDH
Observe recommendations

### Snapshot (Link)

https://pastebin.com/ametAUL2

**Comment by taherbert** [comment]
I'm not super familiar with how to read the Hekili logs linked, but can say that what's described in the "TLDR" here isn't what sims do or what guides recommend. There are valid times to use fracture when at max souls and it is very common to spam cleave/sunder at zero souls without fracture between.

That isn't to say the addon's recommendations are correct -- just that fracture usage is actually quite nuanced and what might appear to be "incorrect" may actually be the right thing to do.

**Comment by dubudevs** [comment]
> I'm not super familiar with how to read the Hekili logs linked, but can say that what's described in the "TLDR" here isn't what sims do or what guides recommend. There are valid times to use fracture when at max souls and it is very common to spam cleave/sunder at zero souls without fracture between.
> 
> That isn't to say the addon's recommendations are correct -- just that fracture usage is actually quite nuanced and what might appear to be "incorrect" may actually be the right thing to do.

Of course, however fracture cpm is extremely low compared to real players in relevant content because the APL _heavily_ prefers to use felblade for fury and spam 0 soul cleaves since its a sim gain. However, if you check the wowhead and icyveins guides and video guides they recommend to use fracture unless you will overcap souls or fury or need to press a defensive button or you need to stack frailty, or of course if you don't need souls.

It's mostly down to goal, if the goal is to have the addon recommend only max DPS then that is one thing, but if that comes at the expense of being able to do any content then I'd say that defeats the purpose.

**Comment by syrifgit** [comment]
Closing given that the APL will likely be reworked (by the class theorycrafters) for 11.1 anyways,
